### PR TITLE
feat: List the queued replays [PC-13341]

### PR DESCRIPTION
**Pull request by dawidwisn**
## Motivation

Ability to list all queued replays through sloctl.

## Summary

Added new command `sloctl replay list`

## Related changes

https://github.com/nobl9/n9/pull/15630

### CODE REVIEW

**Review by marcinlawnik — APPROVED** [review]

File: internal/replay_list.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	"github.com/spf13/cobra"
+
+	"github.com/nobl9/sloctl/internal/printer"
+)
+
+// AddListCommand returns cobra command list, which allows to list all queued Replays.
+func (r *ReplayCmd) AddListCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List queued Replays",
```
Comment (paulina-n9) [inline internal/replay_list.go:19]:
```suggestion
		Short: "List all Replays",
```

**Comment by paulina-n9** [comment]
I confirm that everything works as expected 👌 Here's how it looks like (using sloctl) for different role bindings:

Zrzut ekranu 2024-10-21 o 09 59 01

File: internal/replay_list.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+		return err
+	}
+
+	var replayQueueList []ReplayQueueItem
+	if err := json.Unmarshal(response, &replayQueueList); err != nil {
+		fmt.Printf("err: %v\n", err)
+	}
+
+	if len(replayQueueList) == 0 {
+		cmd.Println(colorstring.Color("[light_gray]No Replays found[reset]"))
```
Comment (paulina-n9) [inline internal/replay_list.go:57]:
```suggestion
		cmd.Println(colorstring.Color("[light_gray]Replay not found[reset]"))
```

**Pull request merged**